### PR TITLE
Exclude some tests. Reasons:

### PR DESCRIPTION
**Pull request by tatraian**
Tadam!!!
- OpenSSL link error
- Boost build error
- example link error

### CODE REVIEW

**Comment by ruslo** [comment]
@tatraian please add link to logs here too, thanks

**Comment by tatraian** [comment]
Filtering bases on tests:
- https://ci.appveyor.com/project/ingenue/hunter/build/1.0.404
- https://travis-ci.org/ingenue/hunter/builds/126111928

File: .travis.yml [hunk 1]
```diff
@@ -51,6 +51,42 @@ env:
...
+    # In this case the final binary is cannot be linked to OpenSSL with the following error message:
+    # /usr/bin/ld: /home/travis/build/ingenue/hunter/_testing/Hunter/_Base/ebe8903/e97794b/3d05f7a/Install/lib/libssl.a(s3_meth.o): relocation R_X86_64_32 against `.rodata' can not be used when making a shared object; recompile with -fPIC
+    - os: linux
+      env: PROJECT_DIR=examples/websocketpp TOOLCHAIN=sanitize-thread
+
+    # boost build error on OS X
+    - os: osx
+      env: PROJECT_DIR=examples/websocketpp TOOLCHAIN=clang-libstdcxx
+
+    # XCode builder (???) seg fault...
```
Comment (ruslo) [inline .travis.yml]: It's a Travis glitch, restart helps: https://travis-ci.org/ingenue/hunter/jobs/126111944

Comment (tatraian) [inline .travis.yml]: Which one? The example is osx-10-9. Do I re-enable it?

Comment (ruslo) [inline .travis.yml]: > osx-10-9

Yes, looks like it works fine.

**Comment by tatraian** [comment]
Re-enabled

**Pull request merged**